packages/server/src/env.ts

```diff
@@ -4,5 +4,5 @@ export let HOSTS = [''];
 if (process.env.NODE_ENV === 'production') {
   HOSTS = ['https://our.production.domain1.com', 'https://our.production.domain2.com'];
 } else {
-  HOSTS = ['https://localhost'];
+  HOSTS = ['http://localhost:3000'];
```

Comment:
Can this help here? https://stackoverflow.com/questions/44574399/create-react-app-how-to-use-https-instead-of-http

Reply:
Yeah but first we need to enable https on the server. Which means generating a certificate etc. I will look into that another time...

Reply:
but this is local host :-p anyway doesn't matter now